### ISSUE TITLE
[Feature request] UX improvement with 1password agent

### ISSUE THREAD

**Issue by sino1641** [Eugeny/tabby#9388]
**Is your feature request related to a problem? Please describe.**

Yes, I saw some tickets asking for the implementation with 1p agent on Windows.

**Describe the solution you'd like**
I figure out a workaround:
1. Change the identity file path to pubkey in .ssh/config
2. Ignore the private key password prompt here:
    <img width="642" alt="Prompt" src="https://github.com/Eugeny/tabby/assets/13870295/87aaec00-d672-4e01-840a-975af1b88354">

3. The agent will pop up then it should be able to login.

Now the thing is how to improve the UX of this step and if it is possible integrate the setup process into tabby new profile UI.
 
**Describe alternatives you've considered**
Write down the workaround in readme/wiki.

**Additional context**
Non-English speaker, sorry for my poor English.

**Comment by Mikrz** [comment]
I've already raised at least a more generic approach to this in #8942 if you want to follow that one as well.

**Comment by sino1641** [comment]
> I've already raised at least a more generic approach to this in #8942 if you want to follow that one as well.如果您也想遵循该方法，我已经在 #8942 中至少提出了一种更通用的方法。

Yes, I also saw your approach from 1p document, but I would like to ask if there is a way to integrate the setup process into tabby `new profile` UI.
